index.js

```diff
@@ -0,0 +1,194 @@
+import {injectGlobal} from 'emotion';
```

Comment:
I think we should use 'css' method instead 'injectGlobal' because only then we can extend our globals with these styles... 
```js
// global.js - place in your app where global styles reside
import emotionNormalize from 'emotion-normalize';
import {injectGlobal} from 'emotion';

/* eslint-disable no-unused-expressions */
injectGlobal`
${emotionNormalize}

*, *::after, *::before {
  box-sizing: border-box;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
  font-smoothing: antialiased;
}`;
/* eslint-enable */
```

Reply:
It worked when I tested it, but sure, I'll use css :+1: